fuse/readonly/mount_unix.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 // Mount mounts ipfs at a given location, and returns a mount.Mount instance.
-func Mount(ipfs *core.IpfsNode, mountpoint string) (mount.Mount, error) {
+func Mount(ipfs *core.IpfsNode, mountpoint string, allow_other bool) (mount.Mount, error) {
```

Comment:
no worries-- go-ipfs is too complex. also feel free to ask @whyrusleeping and me questions about approaches, we can probably save you lots of reading/thinking time.